cmd/kubeadm/app/cmd/phases/join/controlplaneprepare.go

```diff
@@ -74,8 +74,8 @@ func getControlPlanePreparePhaseFlags() []string {
 
 func newControlPlanePrepareDownloadCertsSubphase() workflow.Phase {
 	return workflow.Phase{
-		Name:  "download-certs",
-		Short: fmt.Sprintf("Download certificates from %s", kubeadmconstants.KubeadmCertsSecret),
+		Name:  "download-certs [api-server-endpoint]",
+		Short: fmt.Sprintf("[EXPERIMENTAL] Downloads certificates shared among control-plane nodes from the %s Secret", kubeadmconstants.KubeadmCertsSecret),
 		Long:  cmdutil.MacroCommandLongDescription,
```

Comment:
`MacroCommandLongDescription` should be removed. It generates wrong and misleading help message.

Reply:
i will do the second pass on join phases and clean this up.

Reply:
Thanks. I missed this one

Reply:
Fixed while rebasing

Reply:
/lgtm